src/wheatley.ts

```diff
@@ -386,6 +387,15 @@ export class Wheatley {
         await this.client.login(config.token);
     }
 
+    async fetch_emoji() {
+        if (this.client.application) {
+            await this.client.application.emojis.fetch();
+            for (const [key, value] of this.client.application.emojis.cache) {
+                this.emoji_map[value.name as keyof typeof this.emoji_map] = `<:${value.identifier}>`;
```

Comment:
This may need to be adjusted to include a snowflake and also handle animated emojis differently

Reply:
It's been a while since I checked, but I think `.identifier` should already take care of all that.